### PR TITLE
FIX Syncrhonize updating new metada from sdc into paas database

### PR DESCRIPTION
**Pull request by jmmovilla**
@hmunfru @jframos

### CODE REVIEW

File: core/src/test/java/com/telefonica/euro_iaas/paasmanager/manager/ProductReleaseManagerImplTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    private ProductRelease productReleaseSDC = null;
+
+    private PaasManagerUser user;
+    private ClaudiaData claudiaData;
+    
+    Metadata metadata_diff = new Metadata("key_diff", "value_diff");
+	    
+	@Override
+    @Before
+    public void setUp() throws Exception {
```
Comment (flopezag) [inline core/src/test/java/com/telefonica/euro_iaas/paasmanager/manager/ProductReleaseManagerImplTest.java]: In this method, I see something wrong with the indentation of the lines.

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/ProductReleaseManagerImpl.java [hunk 2]
```diff
@@ -106,47 +110,37 @@ public ProductRelease loadFromSDCAndCreate(String name, ClaudiaData data) throws
...
             try {
                 productRelease = productReleaseDao.load(name);
-                productRelease = productReleaseDao.update(productRelease);
-
-                boolean isNew;
+               
                 for (Attribute attribute : pRelease.getAttributes()) {
-                    isNew = false;
                     Attribute newAttribute = productRelease.getAttribute(attribute.getKey());
                     if (newAttribute == null) {
```
Comment (jframos) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/ProductReleaseManagerImpl.java]: What if the metadata is not new and the value should be updated?

**Comment by jframos** [comment]
LGTM

**Comment by flopezag** [comment]
LGTM

**Pull request merged**